### PR TITLE
Tidy up pytest-based tests

### PR DESCRIPTION
**Pull request by funbringer**
I took a conservative approach and mostly tried to fix various style issues found by `flake8`. PEP8 formatting style differs from the one defined by rustfmt, but I don't see the problem in that.

### CODE REVIEW

File: test_runner/fixtures/zenith_fixtures.py [hunk 1]
```diff
@@ -358,7 +418,8 @@ def zenith_binpath(base_dir):
 
 @zenfixture
 def pg_distrib_dir(base_dir):
-    """ find the postgress install """
+    """ Find the postgress install. """
```
Comment (hlinnaka) [inline test_runner/fixtures/zenith_fixtures.py:416]: typo

File: test_runner/setup.cfg [hunk 2]
```diff
@@ -0,0 +1,12 @@
+# Just trying to gather linter settings in one file.
```
Comment (hlinnaka) [inline test_runner/setup.cfg:1]: What's the role of all of these different tools? Do we need to run all of them, or are they alternatives to each other?

**Review by hlinnaka — APPROVED** [review]
Looks good to me

File: test_runner/setup.cfg (hunk 2 above)
Comment (funbringer) [reply test_runner/setup.cfg:1]: That's a good question, actually. I use flake8 and yapf most of the time, but python-language-server also calls pyflakes, pycodestyle and god knows what else. I had to duplicate those settings to make these tools agree with each other.

File: test_runner/fixtures/zenith_fixtures.py (hunk 1 above)
Comment (funbringer) [reply test_runner/fixtures/zenith_fixtures.py]: Done. I guess we could run codespell on CI.

**Review by ericseppanen — APPROVED** [review]
I'm no expert on python style, but these look good to me.

**Pull request merged**